src/main/java/com/wire/bots/roman/resources/BroadcastResource.java

```diff
@@ -142,6 +106,79 @@ public Response post(@Context ContainerRequestContext context,
         }
     }
 
+    private void broadcastCall(IncomingMessage message, UUID providerId, UUID broadcastId) {
+        for (UUID botId : botsDAO.getBotIds(providerId)) {
+            broadcast.submit(() -> {
+                try {
+                    final UUID messageId = sender.sendCall(message, botId);
+                    if (messageId != null) {
```

Comment:
what is the situation when this is `null`? what happens?

Reply:
Null happens if Sender is unable to initiate WireClient. This usually happens when the bot is removed from the conversation and cryptobox state is deleted but for some reason Roman still thinks the message should be sent as this bot. What we want to do is to skip this botId. Maybe DB clean up is needed in this case

Reply:
ok, thanks for the clarification!